inbox/mailsync/backends/imap/common.py

```diff
@@ -45,10 +53,12 @@ def local_uids(
     if limit:
         q = q.order_by(desc(ImapUid.msg_uid))
         q = q.limit(bindparam("limit"))
-    q = q.params(account_id=account_id, folder_id=folder_id, limit=limit).yield_per(
-        1000
-    )
```

Comment:
yield per is SQLAlchemy specific to save on RAM with all those extra intermediate objects that SQLAlchemy is creating and is no longer relevant as we skip them.